Coordination Meeting 2021-01-07 (#866)
HSF/hsf.github.io: organization/_posts/2021/2021-01-07-coordination.md

- Caterina Doglioni: LHCb seems covered, I was suggesting a formal/informal "introduction to the new HSF WG conveners" e.g. in weekly meetings of ATLAS / CMS.
- Graeme: also worth looking back at mandates to check whether they are clear enough - we should make clear that we're here to be helpful. **ACTION**
- Marc Paterno: we could write a letter to other experiments as well to let them know they're welcome to join.
    - Graeme: we have good contacts with LHC and Belle-II, DUNE. We could write an email to the various experiments, pointing out the new conveners in 2021, and we can come and talk about HSF in the New Year. **ACTION**

#### Workshops

- Kevin Pedro: Zoom-fatigue problem -> make the workshops more focused on particular topics so that people pick and choose what to attend.
- Graeme: the 4-day workshop could probably have been spread out in 4 different meetings... even if it was focused and had improved based on the May workshop feedback.

---

- Andi Salzburger: on workshop, "complaints" about length of workshops especially late in the year. More topical workshop: should the WGs be doing their individual "workshops"?

> - CD: agree with Andi. If no workshop, will we miss joint workshops with WLCG?

Reviewer: ```suggestion
- CD: agree with Andi. If there is no workshop, will we miss joint workshops with WLCG?
```

    - Liz and Graeme are in the management board - need to also discuss there.
- Andrea Valassi: are we actually managing to get people to talk together even when they go to the same workshop? There is the risk that people only attend their session.
- Andi: if this is "out of the box" of the WGs, we could have joint sessions between WG A and WG B. This could be a working mode. 
- Graeme: in particular we could also talk about I/O matters, which links analysis, frameworks and WLCG.

> - Conclusion from this part: explore more restricted but cross-cutting topics. No appetite for a spring workshop, but keep open option for autumn (TBD with WLCG). **ACTION**

Reviewer: ```suggestion
- Conclusion from this part: explore more restricted but cross-cutting topics. No appetite for a spring workshop, but keep an open option for autumn (TBD with WLCG). **ACTION**
```

#### Training

- Sudhir Malik - Training WG: 

>     - We (training) come from different experiments, so it would be good to understand what we are providing / what are our contributions in terms of simulation etc. Highlighting our success in terms of practical things. Some answer to "What have we achieved". **ACTION**

Reviewer: ```suggestion
    - We (training) come from different experiments, so it would be good to understand what we are providing / what are our contributions in terms of simulation etc. Highlighting our success in terms of practical things. Some answers to "What have we achieved". **ACTION**
```

#### Software Projects

- Pere Mato: in terms of "hosted projects" we can't necessarily do like Apache, but we could aim for doing more. Where we can facilitate common work on software, we / conveners should jump on that.
- Andrea Valassi: within e.g. the Generators WG there is more preparatory work that has helped out being involved in porting Madgraph to GPUs.
- Graeme: in the coordination group we could do more, and see if there are other projects that want to be under the umbrella of HSF. E.g. the Phoenix event display was happy in being an HSF project, even the banner home could be a useful thing.

#### Post-Community White Paper

- Eduardo: have you looked back at the community whitepaper? 
- Graeme: It takes a lot of work to do full reviews, but we should take a look at that and see how things are going (we did some of it for the LHCC document). 
- Eduardo: not at all suggesting to prepare a formal document. The (informal) exercise will be useful for us in any case.
  
## News, general matters, announcements

### HDR Institute - Letter of Collaboration

> We have been contacted by Mark Neubauer and Shih-Chieh Hsu asking for a letter of intention to collborate if their NSF proposal for *HDR Institute: Accelerated AI Algorithms for Data-Driven Discovery* is funded. Proposal:

Reviewer: ```suggestion
We have been contacted by Mark Neubauer and Shih-Chieh Hsu asking for a letter of intention to collaborate if their NSF proposal for *HDR Institute: Accelerated AI Algorithms for Data-Driven Discovery* is funded. Proposal:
```

> > Graeme Stewart has agreed to coordinate the collaboration between the HEP Software Foundation (HSF) and A3D3. This collaboration will include AI efforts in event reconstruction and software triggering, and cohosting training and community events of mutual interest.

Reviewer: ```suggestion
> Graeme Stewart has agreed to coordinate the collaboration between the HEP Software Foundation (HSF) and A3D3. This collaboration will include AI efforts in event reconstruction and software triggering, and co-hosting training and community events of mutual interest.
```

A summary of the proposal and draft letter (it's the NSF standard text) are attached to Indico.

> The coordination group are favourable. Please give any feedback or comment by 11 January.

Reviewer: ```suggestion
The coordination group is favourable. Please give any feedback or comment by 11 January.
```
